Makefile

```diff
@@ -15,10 +15,10 @@ LD_FLAGS	= -L /usr/local/lib -l $(GTEST_LIB) -l pthread
 all : test bench
 
 $(TESTDIR)/%.o : $(TESTDIR)/%.cpp $(SRCS)
-		$(CXX) -march=icelake-client -O3 $(CXXFLAGS) -c $< -o $@
+		$(CXX) -march=icelake-client -O3 $(CXXFLAGS) -l $(GTEST_LIB) -c $< -o $@
```

Comment:
Why is this needed? This is only building object files and doesn't need any linking to done.